gax-grpc/src/main/java/com/google/api/gax/grpc/ChannelPool.java

```diff
@@ -90,8 +111,20 @@ static ChannelPool createRefreshing(
    * @param channels a List of channels to pool.
    */
   private ChannelPool(List<ManagedChannel> channels) {
+    this(channels, null);
+  }
```

Comment:
I think you should drop this constructor and have the factory method call the variant below directly.